data/src/main/java/com/microsoft/azure/kusto/data/http/SimpleProxyPlanner.java

```diff
@@ -0,0 +1,48 @@
+package com.microsoft.azure.kusto.data.http;
+
+import org.apache.http.HttpHost;
+import org.apache.http.HttpRequest;
+import org.apache.http.impl.conn.DefaultRoutePlanner;
+import org.apache.http.impl.conn.DefaultSchemePortResolver;
+import org.apache.http.protocol.HttpContext;
+
+public class SimpleProxyPlanner extends DefaultRoutePlanner {
```

Comment:
So you don't need the class anymore - at least not here, since it is only used it the test.

Even in the test you can replace it with a lambda (java can automatically accept lambdas for 1-function interfaces).

Might be worth to keep as an example somewhere, but not in the middle of the code.